arduinoSrc/WebServer/WebServer.ino

```diff
@@ -70,8 +82,17 @@ void setup(void)
     Serial.println("HTTP server started");
 }
 
+//when the sensor detected the distance from the obstacle is less than 15cm
+//the car stops
+void obstacleAvoid() {
+    if (front.getDistance() <= MIN_OBSTACLE_DISTANCE && front.getDistance() > 0) {
```

Comment:
This will trigger two different measurements, so it's both unnecessarily slow and potentially wrong if the two calls return different results.